### DISCUSSION TITLE
Is there a howto for Setup of a pico with a 3.5" TFT display with touch

### DISCUSSION THREAD

**Discussion by MeghansUncle2** [Bodmer/TFT_eSPI#2588] [Compatible displays and setup files]
Hello, I am very new at using GIT, PICO, TFTs, and programming in MicroPython.  I am hoping that the Bodmer /TFT_eSPI drivers will work with my hardware/software.

I have:
 1) PICO W  
 2) 3.5" TFT display with touch that uses the ILI9486 display IC and XPT2046 touch controller

Forgive me if I've left out any important details.

I am going to be developing with Micropython if the Bodmer /TFT_eSPI drivers will work in this environment.

I have been reading the comments from Bodmer and it looks like the screen and pico are supported  but I may be misunderstanding.

So please answer whether what I want to do is possible with the driver Bodmer /TFT_eSPI, and if so, Please point to any howto for getting (and compiling?) and then installing the driver.  I am not new to programming.  Once I get a head start I think I'll be fine, but I don't see how to perform the steps needed to use this driver.

Thanks in advance for any help,
MeghansUncle2

**Comment by Bodmer** [comment]
TFT_eSPI is an Arduino library. Adaption for use with Micropython may be possible but this is not something I can help with.

**Reply by MeghansUncle2** [reply]
Thanks for the response.  I will try using Arduino code platform.  I am really new to this, does the code you have simply need to be included in the source for the program or is there a compiled component, or both?  I wil be trying the RP2040 code.

**Reply by Bodmer** [reply]
It is a library with examples, you add the library using the Arduino IDE library manager.

**Comment by MeghansUncle2** [comment]
@Bodmer 

I am writing a driver in Micropython for this.  Thanks for all your work on this.  the driver init and a few other things were causing me fits.  Your code is allowing me to see what's happening clearly.  I will gladly provide this code after I test it if you want to include it in your repository.  It's for the ILI9486 with XPT2046 on the PICO only and it's not as well organized as your is. 

Edit,  I noted the use of BGR order in this, and other similar drivers.  Why not use RGB?  I know there must be a reason.

**Reply by badger200** [reply]
Hi I’m using an ILI9486 16-bit parallel with a Pico and I’d love to have a MicroPython version!
